vllm/entrypoints/tool_server.py

```diff
@@ -14,6 +14,10 @@
 if TYPE_CHECKING:
     from mcp.types import ListToolsResult
 
+# tools (not namespaces) which description should be excluded from the prompt
+NO_DESCRIPTION_TOOLS = {"cleanup_session"}
+NO_TOOL_DESCRIPTION_NAMESPACES = {"python"}
```

Comment:
can we avoid this hardcode to python tool?

Reply:
If we get it from MCP in anyway we need to handle it. But let me try to add to envs as well